config.proto

```diff
@@ -92,3 +95,9 @@ enum PropagationFormat {
     // see https://www.w3.org/TR/trace-context/
     TRACECONTEXT = 1;
 }
+
+// IgnoreHeuristics represents the heuristics used to ignore certain calls from tracing
+message IgnoreHeuristics {
+    // url_regex defines the regular expression for the request path to be ignored
```

Comment:
Is the regex evaluated against the full URL or just path part? 

An example of the regex would be helpful.

Reply:
The regex is evaluated against the path part only (without query params)
eg: 
```
      - "\/echo\/live"                     # matches /echo/live
      - "\/user\/.*\/cart".                # matches /user/name/cart
      - "\/user\/[0-9]*\/cart".       # matches /user/10/cart
```